flask_wtf/file.py

```diff
@@ -1,11 +1,15 @@
 import warnings
-from collections import Iterable
 
 from werkzeug.datastructures import FileStorage
 from wtforms import FileField as _FileField
 from wtforms.validators import DataRequired, StopValidation
 
 from ._compat import FlaskWTFDeprecationWarning
+from ._compat import PY2
+if PY2:
```

Comment:
```
try:
    from collections import abc
except ImportError:
    import collections as abc
```

should go in `_compat` and that's what should be imported.

Reply:
Thank you - looks so much nicer!